progenitor-client/src/progenitor_client.rs

```diff
@@ -416,3 +416,32 @@ impl<E> RequestBuilderExt<E> for RequestBuilder {
             })?))
     }
 }
+
+/// Returns a string from any json value which is expected
+/// to work as `multipart/form-data` text. In particular, a
+/// [`serde_json::Value::String`] remains unquoted.
+///
+/// # Example
+/// ```
+/// # use progenitor_client::to_form_string;
+/// use serde_json::json;
+/// assert_eq!(to_form_string::<()>(&json!("a")).unwrap(), "a");
+/// assert_eq!(to_form_string::<()>(&json!(101)).unwrap(), "101");
+/// assert_eq!(
+///  to_form_string::<()>(&json!({"a": 202})).unwrap(),
+///  "{\"a\":202}"
+/// );
+/// ```
+#[doc(hidden)]
+pub fn to_form_string<E>(
+    value: &serde_json::Value,
+) -> Result<String, Error<E>> {
+    match value.as_str() {
+        // Take unquoted String Value
+        Some(v) => Ok(v.to_string()),
+        // otherwise convert to quoted json
+        None => serde_json::to_string(value).map_err(|_| {
```

Comment:
doesn't this make `serde_json` into a mandatory dependency for generated crates?